fetch-uj-records/fetch-uj-records_test.go

```diff
@@ -0,0 +1,64 @@
+package main
+
+import (
+	"bytes"
+	_ "embed"
+	"fmt"
+	"os"
+	"os/exec"
+	"testing"
+
+	"github.com/redhat-appstudio/segment-bridge.git/containerfixture"
+	"github.com/redhat-appstudio/segment-bridge.git/splunk"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	scriptPath   = "../scripts/fetch-uj-records.sh"
+	filePathPass = "./fetch-uj-recordsPass"
+)
+
+func compareOutputs(output []byte, filePath string) bool {
+	/*Recieves the script's output and compares it
+	with a file containing the expected output*/
+	fileOutput, err := os.ReadFile(filePath)
+	if err != nil {
+		fmt.Printf("error: %s\n", err.Error())
+		return false
+	}
+	return bytes.Equal(fileOutput, output)
+}
+
+func runAndValidateScript(t *testing.T, filePath, scriptPath string) bool {
+	/*Returns true if a given scripts runs successfully
+	and it's output is equal to a given file's content*/
```

Comment:
I think the test should be self-explanatory without docstrings. if you don't think it is clear enough, you should how to make it more readable, rather than adding explanation. 
Apply this to all other docstrings in the test files.